Homework_15/homework_15.py

```diff
@@ -0,0 +1,24 @@
+class Rhombus:
```

Comment:
Why do you push again homework 15 file with homework solution into PR with 16 homework solution? Please remove it.